runtime/devnet/src/extensions/v0/assets.rs

```diff
@@ -49,27 +49,33 @@ pub(crate) fn construct_assets_key(
 pub(crate) fn construct_assets_call(
 	call_index: u8,
 	params: Vec<u8>,
-) -> Result<pallet_assets::Call<Runtime, TrustBackedAssetsInstance>, DispatchError> {
+	// ) -> Result<pallet_assets::Call<Runtime, TrustBackedAssetsInstance>, DispatchError> {
+) -> Result<pallet_fungibles::Call<Runtime>, DispatchError> {
 	match call_index {
 		9 => {
```

Comment:
Pallet id doesnt match but yeah

Reply:
Yes this was simply to see if the tests were passing